_sass/includes/_footer.scss

```diff
@@ -15,6 +15,10 @@
             a:hover {
                 color: darken($link-color, 15%);
             }
+            //Added for Issue#10
```

Comment:
Instead of adding comments to code tracking why a certain change was done, we can rely on git (`git blame`) to figure this out. That will ensure our code is clean. :)